### PR TITLE
794 fixed rss samlow

### LINKED ISSUE #794: RSS feed is not updating
## What behavior were you expecting?

RSS feed shows current articles

## What actually happened?

RSS feed contains no items

## Steps to reproduce

Open https://www.ubyssey.ca/rss/

Sorry, unsure if this is caused by the theme or dispatch itself.

### PR DESCRIPTION
**Pull request by SamuelmdLow**
I redid my changes to the rss feed

I put them at 

/rss

/rss/section

/authors/author-slug/rss

### CODE REVIEW

File: ubyssey/views/feed.py [hunk 1]
```diff
@@ -1,68 +1,132 @@
...
+        if self.feed['ttl'] is not None:
+            handler.addQuickElement("ttl", self.feed['ttl'])
+        if self.feed['image_url'] is not None:
+            handler.startElement('image',{})
+            handler.addQuickElement("url", self.feed['image_url'])
+            if self.feed['image_title'] is not None:
+                handler.addQuickElement("title", self.feed['image_title'])
+            if self.feed['image_link'] is not None:
+                handler.addQuickElement("link", self.feed['image_link'])
+            handler.endElement("image")
```
Comment (FireBoyAJ24) [inline ubyssey/views/feed.py]: Formatting: Can we space these lines of code

**Review by FireBoyAJ24 — COMMENTED** [review]
How would a user access he RSS feed of Ubyssey?

**Comment by SamuelmdLow** [comment]
> @FireBoyAJ24  How would a user access he RSS feed of Ubyssey?

I'm planning on adding links to the rss feeds in my redesign branches so that there will be less merge conflicts

**Review by brittkhat — APPROVED** [review]
lgtm

**Pull request merged**